docs/src/modules/components/DiamondSponsors.js

```diff
@@ -19,136 +20,171 @@ export default function DiamondSponsors() {
+              background: `linear-gradient(180deg, rgba(255, 255, 255, 0) 0%, ${
+                (theme.vars || theme).palette.primaryDark[900]
+              } 100%)`,
+            }),
+        ]}
+      />
+      <Box
         sx={[
           (theme) => ({
-            '& a': {
-              width: '100%',
-              height: 52,
-              display: 'flex',
-              alignItems: 'center',
-              justifyContent: 'center',
-              boxSizing: 'border-box', // TODO have CssBaseline in the Next.js layout
-              border: `1px solid`,
-              borderColor: 'grey.200',
-              fontSize: theme.typography.pxToRem(14),
-              fontWeight: 700,
-              borderRadius: 1,
-              transition: theme.transitions.create(['color', 'border-color']),
-              '&:hover': {
-                color: 'primary.500',
-                borderColor: 'grey.300',
-                backgroundColor: 'grey.50',
-              },
-            },
+            display: 'flex',
+            flexDirection: 'column',
+            gap: 1,
+            padding: theme.spacing(1, 2, 2, 2),
+            background: '#FFF',
+            borderTop: '1px solid',
+            borderColor: (theme.vars || theme).palette.grey[100],
           }),
           (theme) =>
             theme.applyDarkStyles({
-              '& a': {
-                borderColor: 'primaryDark.700',
-                '&:hover': {
-                  color: 'primary.300',
-                  borderColor: 'primaryDark.600',
-                  backgroundColor: alpha(theme.palette.primaryDark[700], 0.4),
-                },
-              },
+              background: (theme.vars || theme).palette.primaryDark[900],
+              borderColor: (theme.vars || theme).palette.primaryDark[700],
             }),
         ]}
       >
-        <a
-          data-ga-event-category="sponsor"
-          data-ga-event-action="drawer"
-          data-ga-event-label="octopus"
-          href="https://octopus.com/?utm_source=materialui&utm_medium=referral"
-          rel="noopener noreferrer sponsored"
-          target="_blank"
-        >
-          <Box
-            component="img"
-            height="30px"
-            width="137px"
-            src="/static/sponsors/octopus-light.svg"
-            alt="octopus"
-            title="Repeatable, reliable deployments"
-            loading="lazy"
-            sx={(theme) =>
-              theme.applyDarkStyles({
-                content: `url(/static/sponsors/octopus-dark.svg)`,
-              })
-            }
-          />
-        </a>
-        <a
-          data-ga-event-category="sponsor"
-          data-ga-event-action="drawer"
-          data-ga-event-label="doit"
-          href="https://www.doit.com/flexsave/?utm_source=materialui&utm_medium=referral"
-          rel="noopener noreferrer sponsored"
+        <Button
+          component="a"
+          href="/material-ui/discover-more/backers/#diamond"
```

Comment:
This feels better, getting the top-level page context, while keeping diamond above the fold:

```suggestion
          href="/material-ui/discover-more/backers/"
```